Autofocusing functionality is enabled.

lasuax opened the pull request.

lasuax committed:
Autofocus & PropTypes

lasuax committed:
Autofocusing added.

lasuax commented:
@hugomedl Here it is.

fantapop commented:
@hugomedl any plans to merge this one in?

hugo-ns commented on android/src/main/java/com/rngocoder/BroadcastView.java:
```diff
@@ -93,9 +98,16 @@ public void onHostResume() {
-
+        if (goCoder != null && cameraView != null) {
+            if (mAutoFocusDetector == null)
+                mAutoFocusDetector = new GestureDetectorCompat(localContext, new AutoFocusListener(localContext, cameraView));
```
This line fails to compile, the "AutoFocusListener" class can't be find in this PR.  I assume you're using the same from the GoCoderAndroid Sdk example, please include it.

hugo-ns commented on android/src/main/java/com/rngocoder/BroadcastView.java:
```diff
@@ -55,7 +58,9 @@ public String toString() {
     private int sizePreset;
     private WowzaGoCoder goCoder;
-
+    private boolean autoFocus = true;
```
This autoFocus is not being used anywhere, it should be removed.

hugo-ns reviewed (changes requested):
The changes looks good, there's just missing files that it's stopping the project to compile

lasuax committed:
Requested changes

Added the missing AutoFocusListener.java file and removed unused autoFocus boolean.

lasuax committed:
Merge branch 'master' of https://github.com/lasuax/react-native-wowza-gocoder

lasuax reviewed (commented):
Added the missing class. Sorry about that.
